Scripts~/windows/.build-gamelift-server-sdk.ps1

```diff
@@ -4,9 +4,8 @@ $TEMP_PATH="C:\Temp"
 $NUGET_EXE_PATH="$TEMP_PATH\nuget.exe"
```

Comment:
Do we still need this script? If not we should remove it

Reply:
no - removed